src/mod/applications/mod_http_cache/mod_http_cache.c

```diff
@@ -768,13 +771,22 @@ static char *url_cache_get(url_cache_t *cache, http_profile_t *profile, switch_c
 
 		/* download the file */
 		url_cache_unlock(cache, session);
+		before = switch_time_now() / 1000;
 		if (http_get(cache, profile, u, use_mime_ext, event, session) == SWITCH_STATUS_SUCCESS) {
+			unsigned int duration;
+			after = switch_time_now() / 1000;
+			duration = after - before;
```

Comment:
Why not just make this simpler `(switch_time_now() / 1000) - before`. We don't have any usage for `after`

Reply:
Just wanted to use the `after`. Because it named `before`, so people will try to find the `after` in the future.

Reply:
Fixed it. :)